website/source/docs/providers/aws/r/app_cookie_stickiness_policy.html.markdown

```diff
@@ -15,20 +15,20 @@ Provides an application cookie stickiness policy, which allows an ELB to wed its
 ```
 resource "aws_elb" "lb" {
     name = "test-lb"
-	  availability_zones = ["us-east-1a"]
-	  listener {
-	      instance_port = 8000
-		    instance_protocol = "http"
-		    lb_port = 80
-		    lb_protocol = "http"
-	  }
+	availability_zones = ["us-east-1a"]
+	listener {
+	  instance_port = 8000
+	  instance_protocol = "http"
+	  lb_port = 80
+	  lb_protocol = "http"
+    }
 }
 
 resource "aws_app_cookie_stickiness_policy" "foo" {
-	  name = "foo_policy"
-	  load_balancer = "${aws_elb.lb.id}"
-	  lb_port = 80
-	  cookie_name = "MyAppCookie"
+	name = "foo_policy"
```

Comment:
This is however wrong, when I tried applying this example, I got an error:

```
1 error(s) occurred:

* aws_app_cookie_stickiness_policy.foo: Error creating AppCookieStickinessPolicy: ValidationError: Policy name cannot contain characters that are not letters, or digits or the dash.
	status code: 400, request id: c96ca9bd-5eed-11e5-b3c8-1b09bd7b9609
```

It would be nice to add validation for `aws_app_cookie_stickiness_policy.name`, but more importantly the example should be fixed - e.g. by replacing underscore with dash.